### ISSUE TITLE
0.62.0: Contains site-packages/tests directory

### ISSUE THREAD

**Issue by dvzrv** [pennersr/django-allauth#3752]
Hi! :wave: 

When upgrading the package on Arch Linux to 0.62.0 I noticed that you are now adding the tests directory to the global site-packages dir:

```bash
lstar python-django-allauth-0.62.0-1-any.pkg.tar.zst |rg site-packages/tests
drwxr-xr-x root/root         0 2024-04-24 10:47 usr/lib/python3.11/site-packages/tests/
-rw-r--r-- root/root         0 2024-04-24 10:47 usr/lib/python3.11/site-packages/tests/__init__.py
drwxr-xr-x root/root         0 2024-04-24 10:47 usr/lib/python3.11/site-packages/tests/__pycache__/
-rw-r--r-- root/root       160 2024-04-24 10:47 usr/lib/python3.11/site-packages/tests/__pycache__/__init__.cpython-311.opt-1.pyc
-rw-r--r-- root/root       160 2024-04-24 10:47 usr/lib/python3.11/site-packages/tests/__pycache__/__init__.cpython-311.pyc
-rw-r--r-- root/root      8441 2024-04-24 10:47 usr/lib/python3.11/site-packages/tests/__pycache__/settings.cpython-311.opt-1.pyc
-rw-r--r-- root/root      8441 2024-04-24 10:47 usr/lib/python3.11/site-packages/tests/__pycache__/settings.cpython-311.pyc
-rw-r--r-- root/root       339 2024-04-24 10:47 usr/lib/python3.11/site-packages/tests/__pycache__/urls.cpython-311.opt-1.pyc
-rw-r--r-- root/root       339 2024-04-24 10:47 usr/lib/python3.11/site-packages/tests/__pycache__/urls.cpython-311.pyc
-rw-r--r-- root/root      9209 2024-04-24 10:47 usr/lib/python3.11/site-packages/tests/settings.py
-rw-r--r-- root/root        97 2024-04-24 10:47 usr/lib/python3.11/site-packages/tests/urls.py
```

This should not be done as it will conflict in that namespace.

@pennersr they can be ignored similar to the examples: https://github.com/pennersr/django-allauth/blob/85fd652ca28aee8f5fa7aca791869d06ef826bc4/setup.cfg#L66-L67

**Comment by pennersr** [comment]
Thanks for reporting -- fixed via e561e769 and 10ab6b0f

**Comment by pennersr** [comment]
Closed via ed376425

**Comment by pennersr** [comment]
https://codeberg.org/allauth/django-allauth/issues/3752
